pkg/vulnsrc/osv/osv.go

```diff
@@ -0,0 +1,193 @@
+package osv
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"path/filepath"
+	"time"
+
+	bolt "go.etcd.io/bbolt"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/utils"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+	vtypes "github.com/aquasecurity/vuln-list-update/osv"
```

Comment:
Please don't depend on vuln-list-udpate. You can define a new struct having necessary fields only.

Reply:
Done